server/config.ts

```diff
@@ -154,6 +154,7 @@ const config: ServerConfig = {
         Number(process.env.FEATURE_MLS_CONFIG_KEYING_MATERIAL_UPDATE_THRESHOLD),
       PERSIST_TEMPORARY_CLIENTS: process.env.FEATURE_PERSIST_TEMPORARY_CLIENTS != 'false',
       SHOW_LOADING_INFORMATION: process.env.FEATURE_SHOW_LOADING_INFORMATION == 'true',
+      USE_CORE_CRYPTO: process.env.FEATURE_USE_CORE_CRYPTO == 'true',
     },
     MAX_GROUP_PARTICIPANTS: (process.env.MAX_GROUP_PARTICIPANTS && Number(process.env.MAX_GROUP_PARTICIPANTS)) || 500,
```

Comment:
This should not be considered a magic number, since it's assigned to an object properties (so it has a name). 
This magic number error should not be raised